Python/pdf_reorder/README.md

```diff
@@ -0,0 +1,32 @@
+
+## A pdf reorder or Rearranging Script
+
+#### I used the pdfrw library 
+It is built purely in python
+
+A more efficient,faster and non python built alternatives is PyMuPDF.
+The PyMuPDF library can dependency issues hence we will go for a more simpler **pdfrw library
+
+You can download pdfrw using
+	pip install pdfrw
+	
+[more information on pdfrw on PyPi page ](https://pypi.org/project/pdfrw/#id25)
+
+
+####Now About the script pdf_reorder.py
+
+It takes path of the file as an Input
+Then it takes current page and the page you want it to be on seperate values by a comma ','
+It parses and sorts the input values by the function input_and_parse(n) 
+which takes an input of length of the pdf.
+
+Then using the reorder() function we can get the desired file after rearranging
+
+[See the code here for more information about how the script works](https://github.com/Rahul555-droid/Rotten-Scripts/blob/mohta-rahul's-branch/Python/pdf_reorder/pdf_reorder_.py)
```

Comment:
You are referencing your Local Branch here. Please try to prefer the main branch of the original repository. If your fork gets deleted, this hyperlink will simply die off

Reply:
Ok will do.